assets/js/blocks/product-search/index.tsx

```diff
@@ -95,18 +109,102 @@ registerBlockVariation( 'core/search', {
 			/>
 		),
 	},
-	// @ts-ignore waiting for @types/wordpress__blocks update
-	isActive: ( blockAttributes, variationAttributes ) => {
-		return (
-			blockAttributes.query?.post_type ===
-			variationAttributes.query.postType
-		);
-	},
 	category: 'woocommerce',
 	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
 	description: __(
 		'A search box to allow customers to search for products by keyword.',
 		'woo-gutenberg-products-block'
 	),
-	attributes: PRODUCT_SEARCH_ATTRIBUTES,
+	supports: {
+		align: [ 'wide', 'full' ],
+		inserter: isBlockVariationAvailable ? false : true,
```

Comment:
Nit: if you want, this can be simplied as
```suggestion
		inserter: ! isBlockVariationAvailable,
```

Reply:
Ahh, thank you, great catch!